src/App.js

```diff
@@ -2,8 +2,15 @@ import React, {useEffect, useState, useCallback} from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import styles from "styled-components";
```

Comment:
According to the docs, the proper import is the following, but again, you shouldn't be using it in this assignment really
```suggestion
import styled from "styled-components";
```